src/shared/components/ui/Modal/Modal.js

```diff
@@ -82,13 +85,16 @@ const Modal = forwardRef(
       i18n,
       isComponent,
       component,
-      isSubmit
+      isSubmit,
+      isBodyPadding,
+      headingAlign,
+      footerAlign
     },
     ref
   ) => {
     const renderTitle = () => (
       <div className={styles.modal__header}>
-        {i18n.title && <ModalHeader title={i18n.title} />}
+        {i18n.title && <ModalHeader title={i18n.title} align={headingAlign} />}
 
         {image && <Image src={image} size='auto' height={20} />}
 
```

Comment:
może poprawisz to defaultClassnames tak przy okazji? ;)

Reply:
Zrobione :)